modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/site_settings/site_url.jsp

```diff
@@ -191,7 +193,7 @@ if (privateVirtualHostnames.isEmpty()) {
 				cssClass="lfr-form-row"
 			>
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: